tests/lib/pkgdb.sh

```diff
@@ -475,6 +475,16 @@ distro_install_build_snapd(){
 
         case "$SPREAD_SYSTEM" in
             fedora-*|centos-*)
+                # We need to wait until the man db cache is updated before do daemon-reexec
+                # Otherwise the service fails and the system will be degraded during tests executions
+                for i in $(seq 20); do
+                    man_db_cache_update_service=$(systemctl list-units | grep -E 'run-.*.service' | awk '{print $1;}')
```

Comment:
Perhaps this can be simplified to use `systemctl is-active`?

Reply:
Thanks for reviewing this. I didn't know is-active could take a pattern, I already made the change. 
Thanks for the suggestion.